### DISCUSSION TITLE
How can we disable iqCallee on a client?

### DISCUSSION THREAD

**Discussion by singpolyma** [xmppjs/xmpp.js#994] [Q&A] [answered]
We are using the client in a bit of a low-level way and having xmpp.js auto-reply to iqs that we already replied to is not helpful.  Any way to disable this?

**Comment by sonnyp** [comment, marked as answer]
Are you using it wrong? https://github.com/xmppjs/xmpp.js/tree/main/packages/iq#callee

If you want to disable something; build your own client and remove what you don't want https://github.com/xmppjs/xmpp.js/blob/main/packages/client/index.js
